src/main/java/by/yahor_kulesh/config/ApplicationContext.java

```diff
@@ -20,12 +20,8 @@
 @PropertySource("classpath:application.properties")
 @EnableTransactionManagement
 @EnableAspectJAutoProxy
-public class ApplicationContext {
-
-    @Bean
-    public PlatformTransactionManager transactionManager(DataSource dataSource) {
-        return new DataSourceTransactionManager(dataSource);
-    }
+@EnableJpaRepositories("by.yahor_kulesh")
+public class ApplicationContext implements WebMvcConfigurer {
```

Comment:
when your app is SpringBoot app and the main class annotated with `@SpringBootApplication` is located in root folder, than you do not need additionally specify the following annotations in config class:
@ComponentScan("by.yahor_kulesh")
@PropertySource("classpath:application.properties")
@EnableJpaRepositories("by.yahor_kulesh")

In your custom configuration class keep only those annotations, which are related exactly to this class.
All general annotations, related to the whole app (like `@EnableTransactionManagement` and `@EnableAspectJAutoProxy`) move to App class